test/metabase/legacy_mbql/schema_test.cljc

```diff
@@ -189,3 +189,10 @@
     ::mbql.s/Addable [:relative-datetime -1 :month]
     ::mbql.s/Addable [:interval -2 :month]
     ::mbql.s/+       [:+ [:relative-datetime -1 :month] [:interval -2 :month]]))
+
+(deftest ^:parallel filter-test
+  (are [x] (not (me/humanize (mr/explain ::mbql.s/Filter x)))
```

Comment:
Do we already have QP tests that we can compile and run such expressions? If not it would be good to add some. Also some `lib.convert` tests for conversion to/from pMBQL. Maybe those depend on changes in my branch; if so, I can add them there after rebasing.

Reply:
I think they depend on your branch. All tests with these values will fail until it gets fixed